src/mcp_gateway/audit/logger.py

```
@@ -0,0 +1,100 @@
+import json
+import re
+import sys
+from datetime import UTC, datetime
+from typing import Any, Literal
+
+# シークレット判定用
+_SENSITIVE_KEYS_FULL = {"api_key", "token", "secret", "authorization", "password"}
+_SENSITIVE_KEYS_PREFIX = ("ck_",)
+
+# 値に含まれるシークレットを検知する正規表現
+# Bearer トークン、APIキー(sk-, ck-, ghp_等)、32文字以上の16進数（nonce/hash等）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)